src/VisualStudio/CSharp/Impl/CSharpVSResources.resx

```diff
@@ -655,4 +655,7 @@
   <data name="Collapse_usings_on_file_open" xml:space="preserve">
     <value>Collapse usings on file open</value>
   </data>
+  <data name="Show_new_snippet_experience_experimental" xml:space="preserve">
+    <value>Show new snippet experience (experimental)</value>
```

Comment:
It seems the official name is "Code snippets", I guess we should use it :)
https://docs.microsoft.com/en-us/visualstudio/ide/code-snippets?view=vs-2022